openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -92,20 +92,48 @@ jQuery(function () {
         import(/* webpackChunkName: "editions-table" */ './editions-table')
             .then(module => module.initEditionsTable());
     }
+
+    const addRowButton = document.getElementById('add_row_button');
+    const roles = document.querySelector('#roles');
+    const identifiers = document.querySelector('#identifiers');
+    const classifications = document.querySelector('#classifications');
+    const autocompleteLanguage = document.querySelector('.multi-input-autocomplete--language');
+    const autocompleteWorks = document.querySelector('.multi-input-autocomplete--works');
+    const excerpts = document.getElementById('excerpts');
+    const links = document.getElementById('links');
+
     // conditionally load for user edit page
-    if (document.getElementById('add_row_button')) {
-        import(/* webpackChunkName: "user-website" */ './edit')
-            .then(module => module.initEditRow());
-    }
-    // conditionally load for language autocomplete
-    if (document.querySelector('.multi-input-autocomplete--language')) {
-        import(/* webpackChunkName: "user-website" */ './edit')
-            .then(module => module.initLanguageMultiInputAutocomplete());
-    }
-    // conditionally load for works autocomplete
-    if (document.querySelector('.multi-input-autocomplete--works')) {
+    if (
+        addRowButton || roles || identifiers || classifications ||
+        autocompleteLanguage || autocompleteWorks || excerpts || links
+    ) {
         import(/* webpackChunkName: "user-website" */ './edit')
-            .then(module => module.initWorksMultiInputAutocomplete());
+            .then(module => {
```

Comment:
I merged all these into a single entry point with a single name "user-website"

Before we had two bundles books_edit, user-website and the new one introduced in this patchset. We also made multiple requests for the same module in the same page.

Hope the motivation for this change makes sense!

Reply:
I don't have a good grasp on this part of webpack but the change generally makes sense.